### DISCUSSION TITLE
Applying visx-zoom to a XYChart - issue moving graph outside grid and on top of axises

### DISCUSSION THREAD

**Discussion by nilsgoksor** [airbnb/visx#1243] [Q&A]
I'm struggling to find examples of visx-zoom other than the official one: https://airbnb.io/visx/zoom-i and right now I'm totally stuck on one issue.

When I apply it to my XYChart it's not behaving the desired way. I obviously don't want to be able to move the graph on top of the axises or even outside the graph (as in the picture) but no matter what I try I can't seem to get both the XYChart-axises and a movable graph at the same time.

Do you guys have any example I can follow or any idea what might cause the issue?

<img width="348" alt="Screenshot 2021-06-18 at 08 30 12" src="https://user-images.githubusercontent.com/11962929/122516972-b97d1000-d00f-11eb-9452-b25e764534f1.png">

**Comment by KirbyMcKenzie** [comment]
Hey @nilsgoksor did you happen to find a nice way to handle this?

**Comment by ssbarbee** [comment]
bump
